added linked progress plot

williyamshoe opened the pull request:
added linked vertical line that shows up on both progress plots

williyamshoe committed:
added linked progress plot

sbailey commented on py/surveyqa/summary.py:
```diff
@@ -160,9 +161,31 @@ def last(arr):
+console.log(geometry);
+var data = line_source.data;
+var x = data['x'];
+console.log(x);
```
Please remove the leftover debugging `console.log()` calls.

sbailey commented on py/surveyqa/summary.py:
```diff
@@ -160,9 +161,31 @@ def last(arr):
+  }
+  line_source.change.emit();
+}
+'''
```
Please add a few comment to the above `line_source` and `js` code about what this does and why (target audience: a new collaborator reading this code 2 years from now and wanting to match up the code she is reading to the visual elements on the page).

sbailey reviewed (changes requested):
Overall looks good.  Minor requests inline.

williyamshoe committed:
added comments and removed console.log() calls

williyamshoe commented:
Alright; I removed the debugging logs and added comments to describe what the js code and the line_source did. I also shortened the js code a bit by removing an unnecessary for loop.

sbailey commented:
Looks good; thanks.  Merging now.
